kubernetes/config/samples/classy-vision.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: elastic.pytorch.org/v1alpha1
+kind: ElasticJob
+metadata:
+  name: elasticjob-sample
+spec:
+  rdzvEndpoint: "etcd-service:2379"
+  minReplicas: 1
+  maxReplicas: 4
+  replicaSpecs:
+    Worker:
+      replicas: 3
+      restartPolicy: ExitCode
+      template:
+        apiVersion: v1
+        kind: Pod
+        spec:
+          containers:
+            - name: elasticjob-worker
+              image: torchelastic/examples:0.1.0rc1
+              imagePullPolicy: Always
+              ports:
+                - name: elasticjob-port
+                  containerPort: 10291
+              args:
+                - "s3://<your_script_path>/petctl/<username>/my_job/main.py"
+
+                - --config_file
+                - "/data/classy_vision/resnet50_synthetic_image_classy_config.json"
```

Comment:
Previously you had a EFS mount for the dataset - is this not required anymore?

Reply:
1. We can either add instructions to ask user build the image which add data (~50M) in the container image and then user don't have EFS dependency 
2. We can add volumes and volumesmount back . 

I am ok to any option, want to get some feedbacks from you

Reply:
@Jeffwan @drdarshan  We should provide example for both -- with S3 and EFS. For large datasets EFS or a shared NFS sol will be needed and this is a common enough requirement.

Further, as the torchelastic controller is meant to be cloud agnostic, in the docs we should mention somewhere that you can use S3, EFS or similar services in other clouds. If any special configuration is needed for K8S PV mapping those should be described or provide links to relevant docs on AWS side.

Reply:
> * We can either add instructions to ask user build the image which add data (~50M) in the container image and then user don't have EFS dependency

Hello @Jeffwan - the first option looks like it would be lower friction for the initial operator? Like @chauhang mentions we can add pointers to how to mount EFS as a PV for AWS.